layouts/PostLayout.tsx

```diff
@@ -20,9 +21,41 @@ interface LayoutProps {
   children: ReactNode;
 }
 
+const u = (url = '', params: Record<string, string> = {}) => {
```

Comment:
XXX: hack, may need to be removed or moved to a utils file.

Basically this lets you construct URLs with query parameters with a string and an object:

```js
u("https://foo.com/api/whatever", { thing: "value" }) == "https://foo.com/whatever?thing=value"
```

Reply:
TIL! I'll adapt things to that in a patch.